packages/valory/skills/registration_abci/behaviours.py

```diff
@@ -367,7 +368,12 @@ def async_act(self) -> Generator:
            information of the other validators (agents).
         5. Restart Tendermint to establish the validator network.
         """
-        if not self.params.share_tm_config_on_startup:
+
+        exchange_config = self.params.share_tm_config_on_startup
+        log_message = self.LogMessages.config_sharing.value
+        self.context.logger.info(f"{log_message}: {exchange_config}")
```

Comment:
Was this for debugging purposes or is it actually intended to be merged?

Reply:
I figured it'd be useful. It's not strictly necessary of course.